cross/arm64_ci_script.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+function exit_with_error {
+    set +x
+
+    local errorMessage="$1"
+
+    echo "ERROR: $errorMessage"
+    exit 1
+}
+
+#Exit if input string is empty
+function exit_if_empty {
+    local inputString="$1"
+    local errorMessage="$2"
+
+    if [ -z "$inputString" ]; then
+        exit_with_error "$errorMessage"
+    fi
+}
+
+# Cross builds corefx using Docker image
+function cross_build_native_with_docker {
+    __currentWorkingDirectory=`pwd`
+
+    # choose Docker image
+    __dockerImage=" hseok82/dotnet-buildtools-prereqs:ubuntu-16.04-crossx86-ef0ac75-20175511035548"
+    __dockerEnvironmentVariables="-e ROOTFS_DIR=/crossrootfs/arm64"
+    __runtimeOS="ubuntu.16.04"
+
+    __dockerCmd="docker run ${__dockerEnvironmentVariables} -i --rm -v $__currentWorkingDirectory:/opt/code -w /opt/code $__dockerImage"
+
+    # Cross building corefx with rootfs in Docker
+    __buildNativeCmd="./build-native.sh -buildArch=arm64 -$__buildConfig -- cross"
+
+    $__dockerCmd $__buildNativeCmd
+    sudo chown -R $(id -u -n) ./bin
+}
+
+__buildConfig=
+for arg in "$@"
+do
+    case $arg in
+        --buildConfig=*)
+            __buildConfig="$(echo ${arg#*=} | awk '{print tolower($0)}')"
+            if [[ "$__buildConfig" != "debug" && "$__buildConfig" != "release" ]]; then
+                exit_with_error "--buildConfig can be only Debug or Release" true
```

Comment:
What's the `true` for at the end?

Reply:
Copied code from x86 script had the same extra argument

Reply:
Can it be removed?

Reply:
Done